packages/react-scripts/scripts/eject.js

```diff
@@ -45,44 +47,48 @@ prompt(
 
   var folders = [
     'config',
-    path.join('config', 'jest'),
-    'scripts'
+    'config/jest',
+    'scripts',
+    'scripts/utils',
   ];
 
-  var files = [
-    path.join('config', 'env.js'),
-    path.join('config', 'paths.js'),
-    path.join('config', 'polyfills.js'),
-    path.join('config', 'webpack.config.dev.js'),
-    path.join('config', 'webpack.config.prod.js'),
-    path.join('config', 'jest', 'cssTransform.js'),
-    path.join('config', 'jest', 'fileTransform.js'),
-    path.join('scripts', 'build.js'),
-    path.join('scripts', 'start.js'),
-    path.join('scripts', 'test.js')
-  ];
+  // Make shallow array of files paths
+  var files = folders.reduce(function (files, folder) {
```

Comment:
@Timer btw. Is there is any code convention (besides of eslint)?  Use normal function or arrow?
Right now in [`eject.js`](https://github.com/facebookincubator/create-react-app/blob/3fa5e8e7d7a1b0d004e51a39afc2aa074808aa2a/packages/react-scripts/scripts/eject.js) there are both normal and arrow.